force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -271,11 +319,16 @@ public with sharing class FlowControlLoopsCollections {
      */
     public static Integer findWordInList(String wordToFind, List<String> words) {
         // The variable to store the index of the word
-
+        Integer position = -1;
         // Loop through the list of words
-            // If the current word is the word to find, exit the loop
+        for (Integer i = 0; i < words.size(); i++) {
+            if (wordToFind == words[i]) {
+                position = i;
+                break;
+            }
+        }
 
-        return null; // Replace null with the variable you used to store the result
+        return position; 
```

Comment:
Good job breaking out of the loop on line 327. Students miss that. I tend not to use a variable assignment for problems where all I do is return the value. My solution is below. 

```
 public static Integer findWordInList(String wordToFind, List<String> words) {
     for(Integer i = 0; i < words.size(); i++){
         if(words[i] == wordToFind){
             return i;
         }
     }
        return -1;
    }
```